e2e-tests/portal/cypress/e2e/journeys/admin/create-user/update-a-user.spec.js

```diff
@@ -35,13 +35,20 @@ context('Admin user updates an existing user', () => {
         createUser.role(role).click();
       });
       createUser.username().type(userToUpdate.username);
-      createUser.manualPassword().click();
-      createUser.password().type(userToUpdate.password);
-      createUser.confirmPassword().type(userToUpdate.password);
       createUser.firstname().type(userToUpdate.firstname);
       createUser.surname().type(userToUpdate.surname);
       createUser.bank().select(userToUpdate.bank);
       createUser.createUser().click();
+
+      cy.task('getUserFromDbByEmail', userToUpdate.username).then((user) => {
+        // user sets password
+        resetPassword.visitChangePassword(user.resetPwdToken);
+        changePassword.password().type(userToUpdate.password);
+        changePassword.confirmPassword().type(userToUpdate.password);
+        changePassword.submit().click();
+      });
```

Comment:
We do this so often that it might be worth making a re-usable cypress command for it, what do you think?

Reply:
Agreed, I've made this change